main2.m

```diff
@@ -119,32 +130,35 @@
     %     %save meaninful data, estimated data, meaninful data no offset
     save(strcat('data/',experimentName,'/dataset.mat'),'dataset')
 end
+
 %% Data exploration
 % Plot ftDataNoOffset and/vs estimatedFtData
-for i=4:4
-    %     for i=1:size(input.ftNames,1)
-    FTplots(struct(input.ftNames{i},dataset.ftData.(input.ftNames{i}),strcat('estimated',input.ftNames{i}),dataset.estimatedFtData.(input.ftNames{i})),dataset.time);
-end
+if( scriptOptions.printPlots )
```

Comment:
yes I had that idea, but I was still not fully convinced it would be useful. Now that there is another part based on that it makes sense.